app/models/message.rb

```diff
@@ -153,10 +158,10 @@ def assign_tag(tag)
   end
 
   def all_metadata
-    metadata.merge(template&.metadata || {})
+    metadata&.merge(template&.metadata || {})
```

Comment:
nevieme na vstupe nastavit default `{}` ?

Reply:
To tam mame, ale pri vytvarani draftu sa tam nastavoval nil, trochu som to upravila